package.json

```
@@ -58,9 +58,11 @@
     "@jupyterlab/application": "^3.1.0 || ^4.0.0-alpha.16",
     "@jupyterlab/apputils": "^3.1.0 || ^4.0.0-alpha.16",
     "@jupyterlab/coreutils": "^5.1.0 || ^6.0.0-alpha.16",
+    "@jupyterlab/launcher": "^3.5.2",
+    "@jupyterlab/settingregistry": "^3.5.2",
```

let's be consistent on version constrains and drop unused dependencies

```suggestion
    "@jupyterlab/settingregistry": "^3.1.0 || ^4.0.0-alpha.16",
```